CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Changes
 =======
 
+# 4.2.0 / 2020-02-27
+
+* [FEATURE] Automatically uninstall and then install the agent when trying to downgrade agent version on Windows. See [#690][] [@kbogtob][]
+* [BUGFIX] Set windows installer as sensitive resource and use env var to specify windows user credentials to avoid leaks of credentials in logs. See [#691][] and [#694][] [@julien-lebot][]
```

Comment:
```suggestion
* [BUGFIX] Set Windows installer as sensitive resource and use env var to specify Windows user credentials to avoid leaks of credentials in logs. See [#691][] and [#694][] [@julien-lebot][]
```